modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/behavior/impl/MailActivityBehavior.java

```diff
@@ -362,9 +364,14 @@ protected void setEmailSession(Email email, String mailSessionJndi) {
         }
     }
 
-    protected void setCharset(Email email, String charSetStr) {
+    protected void setCharset(Email email, String charSetStr, String tentantId) {
         if (charset != null) {
             email.setCharset(charSetStr);
+        }else{
```

Comment:
This is a nitpick but please follow project standards for formatting.  For example spaces before and after braces and parentheses.  Thanks.  

There are formatting files for IntelliJ and Eclipse in https://github.com/flowable/flowable-engine/tree/master/ide-settings